test/install.spec.ts

```diff
@@ -3,6 +3,7 @@ import install from '../src/install';
 
 import inquirer from 'inquirer';
 import fs from 'fs-extra';
+// @ts-ignore
```

Comment:
why do we need that comment here?

Reply:
there is no declaration for yarn-install, commented for time being

Reply:
got it